x-pack/performance/journeys_e2e/many_fields_discover.ts

```diff
@@ -24,7 +24,7 @@ export const journey = new Journey({
   })
   .step('Expand the first document', async ({ page }) => {
     const expandButtons = page.locator(subj('docTableExpandToggleColumn'));
-    await expandButtons.first().click();
+    await expandButtons.last().click();
```

Comment:
There was an issue with the popover covering the button 

<img width="1277" alt="Screenshot 2024-09-13 at 12 53 24" src="https://github.com/user-attachments/assets/e17364d7-9470-45f7-aab3-dcf29137238e">

Reply:
Thank you!